wmcorepy3-devtools.spec

```diff
@@ -0,0 +1,24 @@
+### RPM cms wmcorepy3-devtools 1.5
+
+# This is a meta-package to group development tool dependencies
+Requires: yuicompressor py3-coverage py3-pylint py3-nose py3-sphinx py3-cherrypy
+Requires: py3-mox3 py3-future py3-mock py3-retry py3-pep8 py3-memory-profiler py3-pymongo
+
+%prep
+%build
+%install
+
+# Generate dependencies-setup.{sh,csh} so init.{sh,csh} picks full environment.
+rm -rf %i/etc/profile.d
+mkdir -p %i/etc/profile.d
+for tool in $(echo %{requiredtools} | sed -e's|\s+| |;s|^\s+||'); do
+  root=$(echo $tool | tr a-z- A-Z_)_ROOT; eval r=\$$root
+  if [ X"$r" != X ] && [ -r "$r/etc/profile.d/init.sh" ]; then
+    echo "test X\$$root != X || . $r/etc/profile.d/init.sh" >> %i/etc/profile.d/dependencies-setup.sh
+    echo "test X\$?$root = X1 || source $r/etc/profile.d/init.csh" >> %i/etc/profile.d/dependencies-setup.csh
+  fi
+done
```

Comment:
There is no need to generate these dependencies-setup.* files any more. cmsBuild auto generate them. blocks like above can be removed